### PR TITLE
Feature lwdev 5653 project refactor structure

### PR DESCRIPTION
**Pull request by igromyko**

### CODE REVIEW

File: client/Lykke.Service.TemplateFormatter.Client/TemplateFormatterClient.cs [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
 
-        public TemplateFormatterClient(string serviceUrl, ILog log)
+        internal TemplateFormatterClient(string serviceUrl, ILog log)
         {
             if (string.IsNullOrWhiteSpace(serviceUrl))
                 throw new ArgumentException("Value cannot be null or whitespace.", nameof(serviceUrl));
 
             _log = log ?? throw new ArgumentNullException(nameof(log));
-            _service = new EmailFormattingService(new Uri(serviceUrl));
+            _service = new TemplateFormatterAPI(new Uri(serviceUrl));
```
Comment (tarurar) [inline client/Lykke.Service.TemplateFormatter.Client/TemplateFormatterClient.cs:25]: I think you have to dispose this object on TemplateFormatterClient disposal

File: src/Lykke.Service.TemplateFormatter/Models/ErrorResponse.cs [hunk 2]
```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace Lykke.Service.TemplateFormatter.Models
+{
+    public class ErrorResponse
```
Comment (tarurar) [inline src/Lykke.Service.TemplateFormatter/Models/ErrorResponse.cs]: I believe you can use ErrorResponse from Lykke.Common library

File: client/Lykke.Service.TemplateFormatter.Client/TemplateFormatterClient.cs (hunk 1 above)
Comment (igromyko) [reply client/Lykke.Service.TemplateFormatter.Client/TemplateFormatterClient.cs:25]: implemented

File: src/Lykke.Service.TemplateFormatter/Models/ErrorResponse.cs (hunk 2 above)
Comment (igromyko) [reply src/Lykke.Service.TemplateFormatter/Models/ErrorResponse.cs]: Seems You are right.
I deleted 2 instances of ErrorResponse and 2 instances of IsAliveResponse from this project

**Review by tarurar — APPROVED** [review]

**Pull request merged**